flexget/api/app.py

```diff
@@ -95,7 +112,12 @@ class API(RestxAPI):
       - methods to auto document and handle :class:`ApiError` responses
     """
 
-    def validate(self, model, schema_override=None, description=None):
+    def validate(
+        self,
+        model: Model,
+        schema_override: Dict[str, List[Dict[str, str]]] = None,
+        description=None
+    ):
```

Comment:
should this return `None`?

Reply:
It returns a decorator, not None.

Type annotations on decorators are still tricky (they're working on it iirc), and I only added annotations on things I was fairly confident I was correct, as to minimise the mental effort needed from the reviewers, therefore this was left (intentionally) untyped.